sessions/recap-project-1/quiz-app/README.md

```diff
@@ -65,3 +66,30 @@ Create the profile page in the same way you created the other two pages.
 - Link the pages via the anchor elements.
 
 > 💡 The counter and switch don't have to work for now, we will implement this later.
+
+
+
+## 4. (Recommended) Structure 
+### Separate the Quiz App CSS code into different files
+The file name should match the name of the
+component.
+
+Move all styles that are used across multiple components to a `global.css` file. Your main CSS file (e.g. `styles.css`) should have several `@import` statements.
+
+The file structure might look something like this:
+
+```
+quiz-app
+├── components
+│   ├── button.css
+│   ├── card.css
+│   ├── header.css
+│   └── navigation.css
+├── global.css
+├── styles.css
```

Comment:
Is there a need for both `styles.css` and `global.css`?

Reply:
Right, I have used parts of this from here: https://github.com/spicedacademy/fs-new-syllabus/pull/46/files which was located in css-structure challenge, but removed since they haven't started with project 1 at that point...
I think it's fine to have both` styles.css `and `global.css`.